lib/module.go

```diff
@@ -132,13 +132,13 @@ func (l Modules) indexByPath() map[string]*Module {
 	return q
 }
 
-// expandRequiredByDependencies takes a list of Modules and
+// ExpandRequiredByDependencies takes a list of Modules and
 // returns a new list of Modules including the ones in their
 // requiredBy (see below) dependency chain.
 // requiredBy dependency
 // Module dependencies are described in two forms requires and requiredBy.
 // If A needs B, then, A requires B and B is requiredBy A.
-func (l Modules) expandRequiredByDependencies() (Modules, error) {
+func (l Modules) ExpandRequiredByDependencies() (Modules, error) {
```

Comment:
Is it essential to export these functions? Idea was to keep the exported interface to what's described in `system` 😉.